shims/nimble

```diff
@@ -2,7 +2,7 @@
 
 set -ueo pipefail
 
-ASDF_BIN="${ASDF_DATA_DIR:-${ASDF_DIR:-$HOME/.asdf}}/bin/"
+ASDF_BIN="${ASDF_DIR:-${ASDF_DATA_DIR:-$HOME/.asdf}}/bin/"
```

Comment:
You're right that ASDF_DIR should be first. I think we can just remove ASDF_DATA_DIR.

```suggestion
ASDF_BIN="${ASDF_DIR:-$HOME/.asdf}/bin/"
```